test/utils.js

```diff
@@ -30,9 +30,10 @@ export const runQuery = (query: string, rootValue: ?any = { accessToken: null, u
  *
  * @see runQuery
  */
-export const runAuthenticatedQuery = (query: string, userID: string = "user-42") => {
-  return runQuery(query, {
-    accessToken: "secret",
-    userID,
-  })
+export const runAuthenticatedQuery = (query: string, rootValue: ?any = {}) => {
+  const authenticatedRootValue = rootValue
+  authenticatedRootValue.accessToken = "secret"
+  authenticatedRootValue.userID = "user-42"
+
+  return runQuery(query, authenticatedRootValue)
```

Comment:
Oops, forgot to take this advice - I'll get another PR in that does this because it's what I really wanted to do, I just didn't know the syntax. ❤️